dockerfiles/alpine/Dockerfile

```diff
@@ -39,23 +48,25 @@ RUN echo "fn main() {println!(\"if you see this, the build broke\")}" > bob/src/
   && echo "fn main() {println!(\"if you see this, the build broke\")}" > bob-apps/bin/brt.rs \
   && echo "fn main() {println!(\"if you see this, the build broke\")}" > bob-apps/bin/bobt.rs \
   && echo "fn main() {println!(\"if you see this, the build broke\")}" > bob-access/src/lib.rs \
-  && cargo build --release --target=x86_64-unknown-linux-musl
+  && cargo build --profile=${BUILD_PROFILE} --target=x86_64-unknown-linux-musl
 
 
 # separate stage for proto build
 RUN echo "fn main() {println!(\"if you see this, the build broke\")} pub mod grpc {include!(\"bob_storage.rs\");}" > bob-grpc/src/lib.rs \
   && mkdir -p bob-grpc/proto
+RUN rm -f bob-grpc/src/bob_storage.rs
 COPY bob-grpc/proto/* bob-grpc/proto/
 COPY bob-grpc/build.rs bob-grpc/build.rs
-RUN cargo build --release --target=x86_64-unknown-linux-musl \
-  && rm -f target/x86_64-unknown-linux-musl/release/deps/bob* \
-  && rm -f target/x86_64-unknown-linux-musl/release/deps/libbob* 
+RUN cargo build --profile=${BUILD_PROFILE} --target=x86_64-unknown-linux-musl \
+  && rm -f target/x86_64-unknown-linux-musl/$(cat ./build_profile_dir)/deps/bob* \
+  && rm -f target/x86_64-unknown-linux-musl/$(cat ./build_profile_dir)/deps/libbob*
 
 # final build
 COPY . .
 ENV BOB_KEY_SIZE=${KEY_SIZE}
 ENV BOB_COMMIT_HASH=${COMMIT_HASH}
-RUN cargo build --release --target=x86_64-unknown-linux-musl
+RUN cargo build --profile=${BUILD_PROFILE} --target=x86_64-unknown-linux-musl
+RUN mkdir /usr/src/bob/target/build_output && mv /usr/src/bob/target/x86_64-unknown-linux-musl/$(cat ./build_profile_dir)/bobd /usr/src/bob/target/build_output/bobd
```

Comment:
Please, combine this 2 `RUN` into one with `&&` operator. Also, place every command on the separate line:
```
RUN cmd1 \
  && cmd2 \
  && cmd3
```

Every new `RUN` command in docker file creates a new layer, which gives a size and performance overhead